bin/latex.py

```diff
@@ -49,9 +49,10 @@ def tex_escape(text):
     if has_newline:
         text = text[:-1]
     text = regex.sub(lambda match: conv[match.group()], text)
+    # Escape multiple spaces separately
+    text = re.sub('  +', lambda m: '\\phantom{' + ('.' * len(m.group())) + '}', text)
     # Escape leading spaces separately
-    regex = re.compile('^ ')
-    text = regex.sub('\\\\phantom{.}', text)
+    text = re.compile('^ ', re.MULTILINE).sub('\\\\phantom{.}', text)
```

Comment:
Maybe the re.compile could be extracted to a const global here, but since it's only used once (/few times) per invocation, and actually not invoked most of the time (eg `bt run`), it's probably better as is.

Reply:
I agree with you. It would be a trade-off between code readability and efficiency. Since this `tex_escape` function is "only" called once for every sample file (my guess would be around 42 times per invocation of `bt pdf` in an average contest), I also prefer to keep the regexes in-lined for code readability :slightly_smiling_face: 

(also, rewriting this right now wouldn't make much sense if we're planning to use Kattis' approach with `lstinputlisting`s later :stuck_out_tongue:)